MathLib/LinAlg/PETSc/PETScVector.h

```diff
@@ -155,14 +155,17 @@ class PETScVector
             VecSetValues(_v, e_idxs.size(), &e_idxs[0], &sub_vec[0], INSERT_VALUES);
         }
 
-        //! Get several entries
+        /// Set local accessible vector in order to get entries.
+        /// Call this before call operator[] or get(...).
```

Comment:
With the current change, this enables to fetch either all entries of a partition or all entries of global vector.